src/components/analytics/index.ts

```diff
@@ -32,23 +32,24 @@ export function outboundLink(
 export function sendTiming(timingCategory: any, timingVar: any, timingValue: any, timingLabel: any) {
   return googleAnalytics.gaCommandSendTiming(timingCategory, timingVar, timingValue, timingLabel)
 }
-
-if (typeof GOOGLE_ANALYTICS_ID === 'string') {
-  googleAnalytics.initialize(GOOGLE_ANALYTICS_ID, {
-    gaOptions: {
-      storage: 'none',
-      storeGac: false,
-      clientId: storedClientId ?? undefined,
-    },
-  })
-  googleAnalytics.set({
-    anonymizeIp: true,
-    customBrowserType: !isMobile
-      ? 'desktop'
-      : 'web3' in window || 'ethereum' in window
-      ? 'mobileWeb3'
-      : 'mobileRegular',
-  })
+if (process.env.NODE_ENV === 'production') {
+  if (typeof GOOGLE_ANALYTICS_ID === 'string') {
```

Comment:
why two nested IFs. Is not the same as doing just an AND?

Reply:
Old implementation, replaced. Two ifs were conceptually separate even though ended up working as an and in this case.